Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs

```diff
@@ -586,22 +586,79 @@ public void When_a_custom_formatter_exists_in_any_loaded_assembly_it_should_over
             result.Should().Be("Property = SomeValue", "it should use my custom formatter");
         }
 
+        [Fact]
+        public void When_a_base_class_has_a_custom_formatter_it_should_override_the_default_formatters()
+        {
+            // Arrange
+            Configuration.Current.ValueFormatterDetectionMode = ValueFormatterDetectionMode.Scan;
+
+            var subject = new SomeClassInheritedFromClassWithCustomFormatterLvl1()
+            {
+                Property = "SomeValue"
+            };
+
+            // Act
+            string result = Formatter.ToString(subject);
+
+            // Assert
+            result.Should().Be("Property = SomeValue", "it should use my custom formatter");
+        }
+
+        [Fact]
+        public void When_there_are_multiple_custom_formatters_it_should_select_a_more_specific_one()
+        {
+            // Arrange
+            Configuration.Current.ValueFormatterDetectionMode = ValueFormatterDetectionMode.Scan;
+
+            var subject = new SomeClassInheritedFromClassWithCustomFormatterLvl2()
+            {
+                Property = "SomeValue"
+            };
+
+            // Act
+            string result = Formatter.ToString(subject);
+
+            // Assert
+            result.Should().Be("Property is SomeValue", "it should use my custom formatter");
+        }
+
+        [Fact]
+        public void When_a_base_class_has_multiple_custom_formatters_it_should_work_the_same_as_for_the_base_class()
+        {
+            // Arrange
+            Configuration.Current.ValueFormatterDetectionMode = ValueFormatterDetectionMode.Scan;
+
+            var subject = new SomeClassInheritedFromClassWithCustomFormatterLvl3()
+            {
+                Property = "SomeValue"
+            };
+
+            // Act
+            string result = Formatter.ToString(subject);
+
+            // Assert
+            result.Should().Be("Property is SomeValue", "it should use my custom formatter");
+        }
+
         [Fact]
         public void When_no_custom_formatter_exists_in_the_specified_assembly_it_should_use_the_default()
```

Comment:
I'm not sure why this method is changed?

Reply:
Reverted it back, thank you...
It was needed because of old cache.

Reply:
Thanks for the clarification 👍